rqd/setup.py

```diff
@@ -61,7 +61,8 @@
         'grpcio',
         'grpcio-tools',
         'psutil',
-        'pywin32; platform_system == "Windows"'
+        'pywin32; platform_system == "Windows"',
```

Comment:
I got an error in python 3.10 with this module, this fixed it (but might not be future proof)
`No module named 'pywintypes'`
```suggestion
        'pywin32==224; platform_system == "Windows"',
```